### PR TITLE
docs: minor edit to the who supports ibis doc

### PR DESCRIPTION
**Pull request by lostmygithubaccount**
## Description of changes

reading the doc back, making some very minor edits

## Issues closed

n/a

### CODE REVIEW

File: docs/concepts/who.qmd [hunk 1]
```diff
@@ -14,15 +14,18 @@ contribute to Ibis.
 
 ## Other companies
 
-Ibis is used by many other companies, with various tools built on top of it.
-These include:
+Ibis is contributed to used by many other companies. Various tools built on
+top of it. Some publicly available include:
```
Comment (cpcloud) [inline docs/concepts/who.qmd:18]: Hm, this looks grammatically incorrect and also now has a sentence fragment in it.

What change were you going for here?

**Review by cpcloud — CHANGES_REQUESTED** [review]
The updated `Other companies` section seems wrong :)

File: docs/concepts/who.qmd (hunk 1 above)
Comment (lostmygithubaccount) [reply docs/concepts/who.qmd:18]: forgot an "and" 

lemme reword

**Comment by lostmygithubaccount** [comment]
and to answer you question -- trying to note that other companies also contribute to Ibis, not just build things on top of it

**Comment by cpcloud** [comment]
> and to answer you question -- trying to note that other companies also contribute to Ibis, not just build things on top of it

Makes sense.

My main concern was with this fragment:

> Various tools built on top of it.

**Review by cpcloud — APPROVED** [review]

**Pull request merged**